Scripts/GUI.js

```diff
@@ -76,14 +76,44 @@ export default function GUI(inModel){
       incidentPhiEnvelope = addEnvelopeControl(ptLightSliderDiv, "φ",
         "slider_incidentPhi", -180, 180, starting_phi);
 
-      let camRotSlider = document.getElementById("slider_camRot");
-      camRotSlider.setAttribute("min", -180);
-      camRotSlider.setAttribute("max", 180);
-      camRotSlider.setAttribute("step", 1);
-      camRotSlider.setAttribute("value", 0);
+      // let camRotSlider = document.getElementById("slider_camRot");
+      // camRotSlider.setAttribute("min", -180);
+      // camRotSlider.setAttribute("max", 180);
+      // camRotSlider.setAttribute("step", 1);
+      // camRotSlider.setAttribute("value", 0);
+    },
+
+    setupButtonCallback = function(button, url) {
+      // Buttons to select predefined BRDFs
+      button.on('click', (event) => {
+        var blob = null;
+        var xhr = new XMLHttpRequest();
+        xhr.open("GET", url);
+        xhr.responseType = "blob";//force the HTTP response, response-type header to be blob
+        xhr.onload = function()
+        {
+          if (this.status == 200) {
```

Comment:
In my humble opinion we should be using `===` instead of  `==` here unless we explicitly want _loose equality_. (I've tested changing to `===` and it works.) A `==`  [converts both values to a common type](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness), which can be powerful but it can also lead to confusing behavior / bugs. 

Here are just two examples of non-intuitive behavior:
  *  `0 == '0'` evaluates to `true`, whereas  `0 === '0'` evaluates to `false`. 
  * `false == '0'` evaluates to `true`, whereas  `false === '0'` evaluates to `false`. 

Because of this, JSHint can actually [be configured to warn](http://jshint.com/docs/options/#eqeqeq) whenever `==` is used. And at least as of a few years ago, [Patrick Cozzi](https://pjcozzi.github.io/#about) was advocating defaulting to `===`. That being said, **I wouldn't be opposed to using `==` where loose equality is advantageous.** 

Here, however, loose equality is not advantageous. We know a priori that `XMLHttpRequest.status` [returns an unsigned short](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/status), which is considered to be a `number` by JS. Which means that unless there is a bug, both operands should be of the same type. If there is a bug, I would argue that the AJAX request should not be considered to have been successful. 

I apologize if you already know about all of this / disagree. I'm just making a bit of a PSA for @DQLin to see as well. I also apologize if you both already knew this / disagree.